src/metabase/driver/common/parameters/values.clj

```diff
@@ -235,6 +235,13 @@
    (number? value) value
    ;; same goes for an instance of CommaSeperated values
    (instance? CommaSeparatedNumbers value) value
+
+   ;; newer operators use vectors as their arguments even if there's only one
+   (vector? value)
```

Comment:
should you use `sequential?` here instead of `vector?`, in case someone else `map`s or `filter`s the value somewhere upstream in the future? I don't think there's any situation where we wouldn't want any kind of sequential value to be treated this way